master/src/main/scala/org/apache/celeborn/service/deploy/master/Master.scala

```diff
@@ -520,7 +520,6 @@ private[celeborn] class Master(
     if (workersSnapShot.contains(workerToRegister)) {
       logWarning(s"Receive RegisterWorker while worker" +
         s" ${workerToRegister.toString()} already exists, re-register.")
-      statusSystem.handleWorkerRemove(host, rpcPort, pushPort, fetchPort, replicatePort, requestId)
```

Comment:
Seems we can remove `handleWorkerRemove()` now.

Reply:
yes, done